test/TestProjects/ResourceClients-LowLevel/ResourceClients-LowLevel.json

```diff
@@ -0,0 +1,142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2021-10-19",
+    "title": "LlcResourceClients"
```

Comment:
Would it be possible to add a "description" field here to describe the expected client/subclient structure generated from this API spec?

Reply:
Done